astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -20,17 +20,49 @@
 from .gcrs import GCRS
 from .cirs import CIRS
 from .hcrs import HCRS
-from .utils import get_jd12, aticq, atciqz, get_cip, prepare_earth_position_vel
+from .utils import get_jd12, get_dut1utc, aticq, atciqz, get_cip, prepare_earth_position_vel
+
+
+MJD_RESOLUTION = 600. / 86400.  # in days (default 10 min)
+JD0 = 2400000.5
+
+
+def _apci13_fast(obstime):
+
+    # calculate Earth parameters for lower time resolution (they change slowly)
+    # only Earth rotation angle needs to be calculated for each time
+
+    jd1, jd2 = get_jd12(obstime, 'tdb')
```

Comment:
The ```erfa``` docs suggest you can use ```tt``` here in place of ```tdb``` without loss of precision. Caching will mean it only makes a difference on first call, but it is significant (20ms vs 1s for 10**5 times)